libs/langchain-google-cloud-sql-pg/src/vectorStore.ts

```diff
@@ -179,6 +185,39 @@ class PostgresVectorStore extends VectorStore {
     )
   }
 
+  static async fromTexts(_texts: string[], _metadatas: object[], _embeddings: Embeddings, _dbConfig: dbConfigArgs): Promise<VectorStore> {
+    const engine = _dbConfig.engine;
+    const tableName = _dbConfig.tableName;
+    const dbConfig = _dbConfig.dbConfig;
+    const vectorStore = await this.create(engine, _embeddings, tableName, dbConfig);
+    const vectors = await _embeddings.embedDocuments(_texts)
+    
+    const documents: Document[] = [];
+
+    for (let i = 0; i < _texts.length; i++) {
+      const doc = new Document({
+        pageContent: _texts[i],
+        metadata: _metadatas[i]
+      })
+      documents.push(doc);
+    } 
+
+    await vectorStore.addVectors(vectors, documents)
```

Comment:
remove vector store and vector creation above and just use `return PostgresVectorStore.fromDocuments(...)`